src/languageclient.rs

```diff
@@ -2074,6 +2074,50 @@ pub trait ILanguageClient: IVim {
         Ok(())
     }
 
+    fn window_handleProgress(&self, params: &Option<Params>) -> Result<()> {
+        info!("Begin {}", NOTIFICATION__WindowProgress);
+        let params: WindowProgressParams = serde_json::from_value(params.clone().to_value())?;
+
+        let busy = !params.done.unwrap_or(false);
```

Comment:
The logic is convoluted here. Simply use `let done = ...` can make the logic clearer.